itsy-bitsy-data-structures.js

```diff
@@ -188,6 +188,8 @@
  *
  * - **Time complexity** refers to the total count of operations an algorithm
  *   will perform given a set of items.
+ *   In other words, it is the total time taken by the program until its
+ *   completion.
```

Comment:
That kinda leaves the wrong impression. It's not a literal measurement of time. It's an estimate of time for arbitrary data. And that estimate is based entirely on the set of operations scaled by the size and shape of the data. Plenty of other things can impact the actual time the program takes.